types/miromico/insight/lifecycle.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$id": "https://akenza.io/miromico/insight/battery.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Battery",
+  "type": "object",
+  "properties": {
+    "batteryConsumption": {
```

Comment:
inconsistent naming with voltage

Reply:
batterConsumption is not equal to batteryVoltage

Reply:
one time you added battery and one time not